.buildkite/build.rayci.yml

```diff
@@ -34,3 +34,13 @@ steps:
       - forge
       - raypy38cu118base
     job_env: forge
+
+  - label: ":tapioca: build: pip-compile dependencies"
+    instance_type: small
+    commands:
+      - rm ./python/requirements_compiled.txt
```

Comment:
is this remove neccessary?

Reply:
It's a bit of a safeguard to make sure we never expose a stale requirements file